### ISSUE TITLE
Extract EMA non-Functional

### ISSUE THREAD

**Issue by mike4llison** [d8ahazard/sd_dreambooth_extension#864]
### Kindly read the entire form below and fill it out with the requested information.

**Please find the following lines in the console and paste them below. _If you do not provide this information, your
issue will be automatically closed._**

`
venv "F:\github\A1111\venv\Scripts\Python.exe"
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Commit hash: 6cff4401824299a983c8e13424018efc347b4a2b
Installing requirements for Web UI

#######################################################################################################
Initializing Dreambooth
If submitting an issue on github, please provide the below text for debugging purposes:

Python revision: 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Dreambooth revision: 9f4d931a319056c537d24669cb950d146d1537b0
SD-WebUI revision: 6cff4401824299a983c8e13424018efc347b4a2b

Checking Dreambooth requirements...
[+] bitsandbytes version 0.35.0 installed.
[+] diffusers version 0.10.2 installed.
[+] transformers version 4.25.1 installed.
[+] xformers version 0.0.14.dev0 installed.
[+] torch version 1.13.1+cu116 installed.
[+] torchvision version 0.14.1+cu116 installed.

#######################################################################################################
`

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?**
Yes
**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?**
Yes
**Have you updated Dreambooth to the latest revision?**
Yes
**Have you updated the Stable-Diffusion-WebUI to the latest version?**
Yes and No, tried it on older versions of A1111 same result
**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this.
Reply 'OK' Below to acknowledge that you did this.**
Yes really, I wouldn't be here if I hadn't exhausted my troubleshooting
**Describe the bug**

As of a few pulls ago, checking `Extract EMA` reported in the model selection window as `Has EMA: False` But when loading in older DB model profiles, it reports as True. I have tried multiple varients of EMA and non EMA models from 1.x-2.1 and still the same effect. I also tried multiple different A1111 installs and and configurations of pytorch installs. Same result.

**Provide logs**

In the db_config.json file it also reports `"has_ema": false,` I even tried changing the config to entry to true. though it reported the change, it had not real effect.

**Environment**

What OS?

Windows X

What GPU are you using?
3090 Tuf

**Screenshots/Config**

` "adamw_weight_decay": 0.01,
    "attention": "default",
    "cache_latents": true,
    "center_crop": true,
    "freeze_clip_normalization": false,
    "clip_skip": 1,
    "concepts_list": [],
    "concepts_path": "",
    "custom_model_name": "",
    "epoch": 0,
    "epoch_pause_frequency": 0,
    "epoch_pause_time": 0,
    "gradient_accumulation_steps": 1,
    "gradient_checkpointing": true,
    "gradient_set_to_none": true,
    "graph_smoothing": 50,
    "half_model": false,
    "train_unfrozen": false,
    "has_ema": false,
    "hflip": false,
    "initial_revision": 0,
    "learning_rate": 5e-06,
    "learning_rate_min": 1e-06,
    "lifetime_revision": 0,
    "lora_learning_rate": 0.0001,
    "lora_model_name": "",
    "lora_rank": 4,
    "lora_txt_learning_rate": 5e-05,
    "lora_txt_weight": 1.0,
    "lora_weight": 1.0,
    "lr_cycles": 1,
    "lr_factor": 0.5,
    "lr_power": 1.0,
    "lr_scale_pos": 0.5,
    "lr_scheduler": "constant_with_warmup",
    "lr_warmup_steps": 0,
    "max_token_length": 75,
    "mixed_precision": "fp16",
    "model_name": "SusV3",
    "model_dir": "F:\\github\\A1111\\models\\dreambooth\\SusV3",
    "model_path": "",
    "num_train_epochs": 100,
    "pad_tokens": true,
    "pretrained_model_name_or_path": "F:\\github\\A1111\\models\\dreambooth\\SusV3\\working",
    "pretrained_vae_name_or_path": "",
    "prior_loss_scale": false,
    "prior_loss_target": 100,
    "prior_loss_weight": 1.0,
    "prior_loss_weight_min": 0.1,
    "resolution": 768,
    "revision": 0,
    "sample_batch_size": 1,
    "sanity_prompt": "",
    "sanity_seed": 420420,
    "save_ckpt_after": true,
    "save_ckpt_cancel": false,
    "save_ckpt_during": true,
    "save_embedding_every": 25,
    "save_lora_after": true,
    "save_lora_cancel": false,
    "save_lora_during": true,
    "save_preview_every": 5,
    "save_safetensors": false,
    "save_state_after": false,
    "save_state_cancel": false,
    "save_state_during": false,
    "scheduler": "ddim",
    "shuffle_tags": false,
    "snapshot": "",
    "src": "M:\\Stable\\Models\\v2-1_768-ema-pruned.ckpt",
    "stop_text_encoder": 1.0,
    "train_batch_size": 1,
    "train_imagic": false,
    "train_unet": true,
    "use_8bit_adam": true,
    "use_concepts": false,
    "use_ema": true,
    "use_lora": false,
    "use_subdir": false,
    "v2": true
}`
Thank you Ethan and D8A for your time and energy you put into this project. I greatly appreciate it.
